packages/cadl-python/test/generated/dev-driven/resiliency/devdriven/_operations/_operations.py

```diff
@@ -170,103 +164,79 @@ def get_model(self, mode: str, **kwargs: Any) -> JSON:
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
-        if response.content:
-            deserialized = response.json()
-        else:
-            deserialized = None
+        deserialized = _deserialize(_models.Product, response.json())
 
         if cls:
-            return cls(pipeline_response, cast(JSON, deserialized), {})
+            return cls(pipeline_response, deserialized, {})
 
-        return cast(JSON, deserialized)
+        return deserialized
 
     @overload
-    def post_model(self, mode: str, input: JSON, *, content_type: str = "application/json", **kwargs: Any) -> JSON:
+    def post_model(
+        self,
+        mode: Union[str, "_models.Mode"],
```

Comment:
I think we can be more consistent with the typing for enums and models. I don't know if we want to switch the ordering to `Union[_models.Mode, str]`, idt it's worth all of the changes we'll make to autorest generated code, but I do think it's worth uniting around using quotes for models and enums. I don't think we need quotes for enums, but i'm not 100% sure

Reply:
done in https://github.com/Azure/autorest.python/pull/1345/commits/996285dac4d72dee33acb6f94072fa5c10785f18 .
I removed quotes around enum annotations in file `_operations.py` and CI looks good.
And I found the quotes are necessary in file `_models.py` since there the "_models" has not been loaded completely.